dependencies.xml

```diff
@@ -40,21 +40,20 @@ Note all install methods after "main" take
     <scipy>1.7</scipy>
     <scikit-learn>1.0</scikit-learn>
     <pandas>1.3</pandas>
-    <!--
-         Note that xarray 0.20 throws this exception:
-         xarray/backends/plugins.py", line 105, in list_engines
-         entrypoints = entry_points().get("xarray.backends", ())
-         AttributeError: 'EntryPoints' object has no attribute 'get'
-         so should be skipped.
-    -->
-    <xarray>0.19</xarray>
+    <!-- Note most versions of xarray work, but some (such as 0.20) don't -->
+    <xarray>2023</xarray>
     <netcdf4>1.5</netcdf4>
     <matplotlib>3.5</matplotlib>
     <statsmodels>0.13</statsmodels>
     <cloudpickle>2.2</cloudpickle>
-    <tensorflow>2.9</tensorflow>
+    <tensorflow os='mac,linux'>2.9</tensorflow>
+    <tensorflow os='windows'>2.6</tensorflow>
+    <!-- Note, tensorflow 2.6 did not require keras, but RAVEN uses keras
+         Most of the other tensorflow versions we have tried do include keras
+         automatically. -->
+    <keras os='windows'>2.4</keras>
```

Comment:
From https://github.com/keras-team/keras, there is a summary about compatibility. They may not update it for while since the new release of Keras is at 2.12 now.

<img width="944" alt="image" src="https://user-images.githubusercontent.com/7321071/217871084-0c5ccea1-43ae-4fb9-a2c5-ec98fa6f0fd2.png">

Reply:
Hm, I'm trying out keras 2.6 and seeing what happens.

Reply:
Keras 2.6 seems to work, so switching.